inlong-sort/sort-connectors/mongodb-cdc/src/main/java/org/apache/inlong/sort/cdc/mongodb/debezium/table/MongoDBConnectorDeserializationSchema.java

```diff
@@ -222,6 +222,18 @@ public void deserialize(SourceRecord record, Collector<RowData> out) throws Exce
         }
     }
 
+    /**
+     * Deserialize the Debezium record, it is represented in Kafka {@link SourceRecord}.
+     *
+     * @param record
```

Comment:
pls add parameter description or just remove it

Reply:
I remove it